delta/plugins/statistics/src/main/resources/elasticsearch/mappings.json

```diff
@@ -0,0 +1,50 @@
+{
+  "properties": {
+    "@id": {
+      "type": "keyword"
+    },
+    "@type": {
+      "type": "keyword"
+    },
+    "properties": {
+      "type": "nested",
```

Comment:
It could be tested later on, but it also says:
> The flattened mapping type should not be used for indexing all document content, as it treats all values as keywords and does not provide full search functionality. The default approach, where each subfield has its own entry in the mappings, works well in the majority of cases.

Reply:
Yes, it can be tested later to see which is better between the nested query and the flattened approach.

With the [reindex api](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html), it should be easy to get an idea quickly